app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Good job adding devise, however, make sure the `:confirmable` is enabled in the user model because it will help us to confirm the mail as it is [required](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Ask%20for%20confirmation%20of%20email.), we will have something like this: 

```ruby
	devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```

Reply:
- `:confirmable` is enabled in the user model.